assets/src/common/helpers/test/__mocks__/amp-block-editor-data.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Mock for amp block editor data to be used in featured image test.
+ *
+ * @type {{featuredImageMinimumHeight: number, featuredImageMinimumWidth: number}}
+ */
+const mockAmpBlockEditorData = {
+	featuredImageMinimumWidth: 1200,
+	featuredImageMinimumHeight: 675,
+};
+
+module.exports = mockAmpBlockEditorData;
```

Comment:
```suggestion
export const mockAmpBlockEditorData = {
	featuredImageMinimumWidth: 1200,
	featuredImageMinimumHeight: 675,
};

```

Reply:
@johnwatkins0 this apparently did not work. See https://github.com/ampproject/amp-wp/pull/5647/checks?check_run_id=1561450704

Reply:
Restored it to how it was e2b842f665ff0183475bf37fe5f2d4bf6d94bb35

Not sure what's going on, but it's too small worry about.